DEVELOPMENT.md

```diff
@@ -0,0 +1,69 @@
+# Developing MXNet
+
+## Build MXNet from source with CMake
+
+It's recommended that you install CMake and Ninja.
+
+Chose the options that you want to compile with in a similar fashion as above. In particular this one is a debug CPU
```

Comment:
Chose --> Choose.
"similar fashion as above..." - what is above? There wasn't any discussion about choosing things yet other than the recommendation of CMake and Ninja. Did some content get moved?